app/Http/Controllers/SupplyController.php

```diff
@@ -0,0 +1,269 @@
+<?php
+
+namespace Adshares\Adserver\Http\Controllers;
+
+use Adshares\Adserver\Models\NetworkCampaign;
+use Adshares\Adserver\Models\NetworkEventLog;
+use Adshares\Adserver\Models\EventLog;
+
+use Adshares\Adserver\Http\GzippedStreamedResponse;
+use Adshares\Adserver\Http\Utils;
+
+use Adshares\Adserver\Services\BannerFinder;
+
+use Adshares\Esc\Esc;
+
+use Illuminate\Http\Request;
+
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\HttpFoundation\StreamedResponse;
+
+// use Adshares\Services\Adselect;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS125)